portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,29 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.setContentType("application/json");
+    List<String> songs = new ArrayList<>();
+    songs.add("9 5 . s o u t h  by J. Cole");
+    songs.add("Alone Again by The Weeknd");
+    songs.add("The Pale Moonlight by Kid Cudi");
+    songs.add("Instant Crush by Daft Punk");
+    String songsJson = new Gson().toJson(songs);
+    response.getWriter().println(songsJson);
```

Comment:
This is probably not the best way to do this. Per [gson docs](https://sites.google.com/site/gson/streaming), it would be more efficient to stream the data into `response`.  The issue with this technique in general is that you have two copies of the data and a redundant copy to make the second one.